core/core_modules/issue.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Creates an issue on github for kassy.
+ *
+ * Written By:
+ * 		Dion Woolley
+ *
+ * License:
+ *		MIT License. All code unless otherwise specified is
+ *		Copyright (c) Dion Woolley and Contributors 2016.
+ */
+
+var command = "issue",
+    githubAPI = "https://api.github.com/repos/mrkno/Kassy/issues",
+    token = 'ZWZkN2ZiMGY2OWViZDlkNmQyZmE0Yzc4ODFjNTkyY2JmNmQxYTljNA==',
```

Comment:
it just needs public_repo,
well that was the minimum I could get it to work with my current token